src/Components/Products/index.js

```diff
@@ -1,10 +1,108 @@
 import React from 'react'
```

Comment:
This file should be at a folder structure named "Screens"

Reply:
Done